pgsql/bin/functions/do_rewind

```diff
@@ -30,6 +30,9 @@ if [[ "$MASTER_SLAVE_SWITCH" == "1" ]]; then
 
     echo ">>>>>> Restoring configs"
     gosu postgres repmgr standby restore-config -D $PGDATA --config-archive-dir=/tmp/repmgr-archive
+else
```

Comment:
There is no straightforward way to determine current timeline without starting postgresql. And in the case when master diverged it won't start at all. From the other side, if replica and master are on the same timeline rewind just does nothing.
Do you have any ideas?